cms/djangoapps/contentstore/views/transcript_settings.py

```diff
@@ -0,0 +1,88 @@
+"""
```

Comment:
I am feeling fully comfortable with transcript_settings.py name?

Reply:
Are you?

Reply:
sorry, typo mistake. I am feeling fully comfortable with transcript_settings.py name? or we will be changing this in the next PR you mentioned below ?